include/Makefile

```diff
@@ -4,7 +4,7 @@ INCS =	xbps.h
 
 .PHONY: all
 all:
-	sed -e "s|@@VERSION@@|${VERSION}|g" ${INCS}.in > ${INCS}
+	@:
```

Comment:
I'm not sure if this is the right solution. Removal of the `all` target would lead to `install` becoming the default one, which is not desirable. I have replaced sed with the "do nothing" `@:` statement instead.